guides/content/developer/customization/s3_storage.md

```diff
@@ -24,6 +24,7 @@ attachment_config = {
   },
 
   storage:        :s3,
+  s3_region:      'us-east-1',
```

Comment:
@otlaitil good catch but it would be better to let the developer change it via env variable eg. 

```
s3_region:      ENV['S3_REGION'],
```